src/common/lru_cache.h

```diff
@@ -279,6 +302,24 @@ bool LRUCache<K, V, KeyTraits, ValueTraits>::Get(const K &key, V *value) {
     return true;
 }
 
+template <typename K,  typename V, typename KeyTraits, typename ValueTraits>
+bool LRUCache<K, V, KeyTraits, ValueTraits>::GetFirst(const V value, K *key) {
```

Comment:
Why does lru provide these operations

Reply:
have reached an agreement that it is not need to edit, because we need traversing the list